packages/aws-cdk-lib/aws-s3-deployment/lib/bucket-deployment.ts

```diff
@@ -112,9 +112,17 @@ export interface BucketDeploymentProps {
    * The number of days that the lambda function's log events are kept in CloudWatch Logs.
    *
    * @default logs.RetentionDays.INFINITE
+   * @deprecated Use logGroup for full control over the custom resource log group
    */
   readonly logRetention?: logs.RetentionDays;
 
+  /**
+   * The Log Group used for logging of events emitted by the custom resource's lambda function.
+   *
+   * @default - a default log group created by AWS Lambda
```

Comment:
if `logRetention` defalt was infinite retention, is that true here also?

Reply:
Yes, the default behavior was to not do anything the log group that the custom resource lambda creates